### PR TITLE
[PRM-dependencyUpgrade]: updated app and plugin dependencies

### PR DESCRIPTION
**Pull request by BackRowOla**

### CODE REVIEW

**Review by LewisMitchell — APPROVED** [review]

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -6,14 +6,14 @@ import sbt._
 object AppDependencies {
 
   val compile = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-frontend-play-28" % "5.3.0",
-    "uk.gov.hmrc"             %% "play-frontend-hmrc"         % "0.64.0-play-28",
+    "uk.gov.hmrc"             %% "bootstrap-frontend-play-28" % "5.12.0",
+    "uk.gov.hmrc"             %% "play-frontend-hmrc"         % "0.94.0-play-28",
     "uk.gov.hmrc"             %% "play-frontend-govuk"        % "0.71.0-play-28",
```
Comment (LewisMitchell) [inline project/AppDependencies.scala:11]: play-frontend-govuk has a new version

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -6,14 +6,14 @@ import sbt._
...
-    "uk.gov.hmrc"             %% "play-language"              % "5.0.0-play-28"
+    "uk.gov.hmrc"             %% "play-language"              % "5.1.0-play-28"
   )
 
   val test = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-test-play-28"         % "5.3.0"            % "test, it",
+    "uk.gov.hmrc"             %% "bootstrap-test-play-28"         % "5.12.0"            % "test, it",
     "org.scalatest"           %% "scalatest"                      % "3.0.9"            % "test, it",
```
Comment (LewisMitchell) [inline project/AppDependencies.scala:17]: scalatest has a new version: 3.2.9

File: project/AppDependencies.scala [hunk 3]
```diff
@@ -6,14 +6,14 @@ import sbt._
...
     "org.jsoup"               %  "jsoup"                          % "1.13.1"           % "test, it",
```
Comment (LewisMitchell) [inline project/AppDependencies.scala:18]: jsoup has a new version: 1.14.2

**Review by LewisMitchell — CHANGES_REQUESTED** [review]
Could you also bump the sbt version to 1.4.9?

File: project/AppDependencies.scala (hunk 2 above)
Comment (BackRowOla) [reply project/AppDependencies.scala:17]: The catalogue doesn't mention an update for scalatest, is there anywhere else that shows dependency updates?

File: project/AppDependencies.scala (hunk 3 above)
Comment (BackRowOla) [reply project/AppDependencies.scala:18]: same as above comment

Comment (LewisMitchell) [reply project/AppDependencies.scala:18]: https://mvnrepository.com/artifact/org.jsoup/jsoup

File: project/AppDependencies.scala (hunk 2 above)
Comment (LewisMitchell) [reply project/AppDependencies.scala:17]: Yeah - it's really annoying to get to.
https://mvnrepository.com/artifact/org.scalatest/scalatest

Comment (BackRowOla) [reply project/AppDependencies.scala:17]: Awesome, thanks

**Review by LewisMitchell — CHANGES_REQUESTED** [review]
play-frontend-hmrc should be able to go to 0.94.0-play-28 safely
play-frontend-govuk should be able to go to 0.84.0-play-28 safely
Could you upgrade SBT to 1.4.9 as well?

**Comment by LewisMitchell** [comment]
Sorry to be a pain ☝️

**Review by LewisMitchell — APPROVED** [review]

**Pull request closed without merging**